src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -5,29 +5,38 @@
 import java.nio.file.Files;
 import java.nio.file.Paths;
 import java.util.Arrays;
+import java.util.Comparator;
 import java.util.List;
 import java.util.stream.Collectors;
 
 public class StreamStudy {
 
+    private static final int MIN_WORD_LENGTH = 12;
+    private static final int MAX_WORDS = 100;
+
     public static long countWords() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+        String contents = Files.readString(Paths
+                .get("src/main/resources/fp/war-and-peace.txt"));
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream()
+                .filter(word -> word.length() > MIN_WORD_LENGTH)
+                .count();
     }
 
     public static void printLongestWordTop100() throws IOException {
-        String contents = new String(Files.readAllBytes(Paths
-                .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
+        String contents = Files.readString(Paths
+                .get("src/main/resources/fp/war-and-peace.txt"));
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream()
+                .filter(word -> word.length() > MIN_WORD_LENGTH)
+                .sorted(Comparator.comparingInt(String::length).reversed())
+                .distinct()
```

Comment:
```suggestion
                .distinct()
                .sorted(Comparator.comparingInt(String::length).reversed())
```
지금은 별 차이 없지만, 데이터가 많을 경우 distinct() 후에 sorted()하는 것과 성능 차이가 있을 수 있어요.